janrain_capture.install

```diff
@@ -103,6 +103,40 @@ function janrain_capture_install() {
 
   // Set the default version
   variable_set('janrain_capture_ver', JANRAIN_CAPTURE_VERSION_DEFAULT);
+
+  // Update users.
+  janrain_capture_email_update_batch();
+
+  // Update mapping.
+  module_load_include('inc', 'janrain_capture_mapping', 'janrain_capture_mapping.admin');
+  $capture_field_options = janrain_capture_mapping_get_capture_field_options();
+  $capture_field_options = array_flip($capture_field_options);
+  $uuid_key = $capture_field_options['UUID'];
+  $map = variable_get('janrain_capture_mapping_map', array());
+  $flag = FALSE;
+  foreach ($map as $key => $item) {
+    if ($item['field'] == 'name') {
+      $flag = $key;
+    }
+  }
+  if ($flag === FALSE) {
+    $map[] = array(
+      'fid' => $uuid_key,
+      'field' => 'name',
+      'column' => 'value'
+    );
+  }
+  else {
+    $map[$flag]['fid'] = $uuid_key;
+  }
+  variable_set('janrain_capture_mapping_map', $map);
+
+  // Update variable.
+  $janrain_capture_fields2 = variable_get('janrain_capture_fields2', array());
+  if(!empty($janrain_capture_fields2)) {
+    $janrain_capture_fields2['capture_no_email'] = 1;
```

Comment:
I would leave this var `capture_no_email` untouched to be keep configuration as is per market needs.